css-layout-classes/index.html

```diff
@@ -28,26 +28,26 @@ <h2>Super Smash Bros. Ultimate</h2>
           online mode was widely criticized.
         </p>
       </div>
-      <div class="column2-image2">
+      <div class="column-one-third">
         <img src="images/yoshi.png" alt="yoshi">
       </div>
     </div>
     <div class="row">
-      <div class="column3-image1">
+      <div class="column-one-quarter">
         <img src="images/kirby.png" alt="Kirby">
       </div>
-      <div>
+      <div class="column-one-quarter">
         <img src="images/fox.png" alt="fox">
       </div>
-      <div>
+      <div class="column-one-quarter">
         <img src="images/pikachu.png" alt="Pikachu">
       </div>
-      <div>
+      <div class="column-one-quarter">
         <img src="images/ness.png" alt="ness">
       </div>
     </div>
     <div class="row">
-      <div class="column4-text1">
+      <div class="column-two-thirds">
         <h2>Donkey Kong</h2>
         <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Maxime debitis ab tenetur odio mollitia dolores placeat perspiciatis velit? Voluptate earum quam sit suscipit omnis iusto cupiditate tempora tempore at aut! Quisquam amet accusantium qui. Tempora vero culpa accusantium laboriosam deserunt facere voluptatum nam dolor neque delectus totam enim, labore quidem.</p>
         <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Temporibus at recusandae eaque illum doloribus tempore sed doloremque! Culpa impedit pariatur iusto cum ducimus dolore, temporibus, nam animi, labore id neque.</p>
```

Comment:
I can;t comment on the line below for some reason, but the `div` holding the donkey kong image should have a `column` class of some sort.